src/components/hosts/utils.ts

```diff
@@ -156,3 +157,9 @@ export const fileSize: typeof filesize = (...args) =>
     .call(null, ...args)
     .toUpperCase()
     .replace(/I/, 'i');
+
+export const findHostById = (hosts: Host[] = [], id: string) =>
+  hosts?.find((host) => host.id === id);
+
+export const unmarshallInventory = (host: Partial<Host> | undefined) =>
```

Comment:
Is `Partial` needed here?

Reply:
Yes this will allow a consumer to call the function without passing the whole host object (which is already quite large)

Reply:
sorry my bad we don't need it